alpha/apps/kaltura/lib/batch2/bcdl/kBusinessPostConvertDL.php

```diff
@@ -315,7 +315,14 @@ public static function handleConvertFinished(BatchJob $dbBatchJob = null, flavor
 			}
 			else
 			{
-			    // mark the context root job as finished only if all conversion jobs are completed
+    			// Export the source
+    			$srcFlavors = assetPeer::retrieveLocalReadyByEntryIdAndFlavorParams($entry->getEntryId(), array(flavorParams::SOURCE_FLAVOR_ID));
+    			if($srcFlavors && count($srcFlavors))
+    			{
```

Comment:
Must validate that asset type is assetType::FLAVOR, since it is possible to have captions asset with falvor param id = 0

Reply:
Done